packages/turf-rhumb-bearing/index.js

```diff
@@ -28,16 +27,69 @@ export default function (start, end, final) {
     if (!start) throw new Error('start point is required');
     if (!end) throw new Error('end point is required');
 
-    var coordsStart = getCoord(start);
-    var coordsEnd = getCoord(end);
-    var origin = new LatLonSpherical(coordsStart[1], coordsStart[0]);
-    var destination = new LatLonSpherical(coordsEnd[1], coordsEnd[0]);
-    var bear360;
+    const coordsStart = getCoord(start);
+    const coordsEnd = getCoord(end);
+    const origin = new LatLon(coordsStart[1], coordsStart[0]);
+    const destination = new LatLon(coordsEnd[1], coordsEnd[0]);
+    let bear360;
```

Comment:
Got to remove that ES6 sugar (`let` & `const`)

Reply:
Meaning we should not use them? Why?

Reply:
Good question... Well the only thing we added from ES6 was the modules, Rollup will only bundle the file but it currently doesn't compile the code to ES5 (We would need to include Babel for that) is that something that's worth doing?

Reply:
@stebogit Going to look into this... if we are already compiling the code, might as well go all the way to ES6 🚀

Reply:
@DenisCarriere I thought this was the idea, to justify all the work we (you) are doing for this change.

Also, only now I realize with this new setup I can't debug the code in PHPStorm (or at least not as easily as before), cause `import/export` break in node... that's a bummer 😞

Reply:
I'm starting to see the same downfalls. 😢 

Well we could always convert the `test.js` back to NodeJS 4 or 6 compatible, that way there's 1 less file that has import/export statements.

We could always explore integrating [`@std/esm`](https://github.com/standard-things/esm) somewhere, maybe that will help?

Reply:
Going wild here: isn't there a way to compile **to** ES6, at least the `import/export` syntax?
That way we could develop (and debug) in node using `require` and the compilation, pre rollup, would take care of that additional feature.

OR, you could just implement `import/export` in Node.js v9.0. I bet with your coffee machine it shouldn't take you that long. 🤣

Reply:
> import/export in Node.js v9.0.

Ideally this would be the ultimate solution.

The goal with this push to ES6 modules is to get ready for that, it obviously requires some HEAVY refactoring (which we are noticing).

Having a `pretest` global script is pretty handy... we can work with that! 👍

Reply:
Currently there's 3 main environments:
- Web UMD (Compile to ES5 UMD format)
- ES6 Modules (and all the other new ES6 features)
- ES5 CommonJS (NodeJS 4 support)

Attempting to support all 3 is difficult... lol! I'm sure we can figure out a low-impact Rollup solution, it's just going to take a bit more tinkering.